etc/profile-a-l/luarocks.profile

```diff
@@ -0,0 +1,67 @@
+# Firejail profile for luarocks
+# Description: LuaRocks is the package manager for the Lua programming language.
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include luarocks.local
+# Persistent global definitions
+include globals.local
+
+# disable blacklist for lua interpreter paths
+# Allow lua (blacklisted by disable-interpreters.inc)
+include allow-lua.inc
+
+include disable-common.inc
+# luarocks can invoke compilers
+#include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+# luarocks is hacky and needs shell access
+#include disable-shell.inc
+include disable-xdg.inc
+
+whitelist ${HOME}/.netrc
+whitelist ${HOME}/.config/pkcs11
+whitelist ${HOME}/.wget-hsts
+whitelist ${HOME}/.cache/luarocks
+whitelist ${HOME}/luarocks/cmd/external
+whitelist ${HOME}/.nix-profile/bin
+whitelist ${HOME}/.luarocks
+whitelist ${HOME}/.config/luarocks
+
+whitelist /usr/share/ca-certificates
+whitelist /usr/share/p11-kit
+whitelist /usr/share/terminfo
+whitelist /usr/share/lua
```

Comment:
Can we use `whitelist-var-common.inc`? `whitelist-run-user-common.inc`? `whitelist-run-common.inc`?

Reply:
1. `whitelist-var-common.inc` -> accessible from user, so the package manager can mess with system logs. luarocks does not need access to /var at all.
2. As I am aware of we dont need access to /run/user either.
3. Neither do we need resolv, systemd, dbus, media, cups or networkmanager settings.